### PR TITLE
User can take different typing tests.

### PR DESCRIPTION
**Pull request by danielduffield**
Created an array with possible typing tests to be selected at random, refactored to make this possible, thereby making the code more flexible.

### CODE REVIEW

**Comment by danielduffield** [comment]
screen shot 2017-06-09 at 11 49 45 pm

File: styles.css [hunk 1]
```diff
@@ -42,10 +42,10 @@ h1 {
 
 .selected {
   font-size: 2em;
-  -webkit-animation: highlight-selected 1.5s infinite; /* Safari 4+ */
-  -moz-animation: highlight-selected 1.5s infinite; /* Fx 5+ */
-  -o-animation: highlight-selected 1.5s infinite; /* Opera 12+ */
-  animation: highlight-selected 1.5s infinite; /* IE 10+, Fx 29+ */
+  -webkit-animation: highlight-selected 1s infinite; /* Safari 4+ */
```
Comment (thebearingedge) [inline styles.css]: In general, you can omit CSS vendor prefixes unless you know for sure that you need to support finicky browsers.
https://caniuse.com/#search=animation
I recommend just sticking with `animation` here.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: styles.css (hunk 1 above)
Comment (thebearingedge) [reply styles.css]: Also, nice touch.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Sweet! Your Pull Request/Issue title could be slightly better though, maybe something like "User can take different typing tests."

**Pull request merged**